### PR TITLE
Op557 option for disabling gravatar

### PR DESCRIPTION
**Pull request by javitonino**
Adds an option to disable Gravatar, specially useful for offline installations (as the timeout if unable to connect to gravatar is 5 minutes).

It is disabled by default on tests (the relevant tests, stub this to the correct value for testing). This (+ #7246) fixes offline tests.

From CartoDB/cartodb-onpremise#557

### CODE REVIEW

**Comment by javitonino** [comment]
And the final piece of the puzzle for offline testing. Can you CR @juanignaciosl?

File: app/models/user.rb [hunk 1]
```diff
@@ -710,13 +712,21 @@ def default_avatar
     "/assets/unversioned/images/avatars/public_dashboard_default_avatar.png"
   end
 
+  def gravatar_enabled?
```
Comment (juanignaciosl) [inline app/models/user.rb]: Maybe you can simplify this. Check `./config/initializers/01_app_config.rb`, method `def self.get_config(*config_chain)`.

**Comment by juanignaciosl** [comment]
Added a suggestion about `gravatar_enabled?`, 👍  after that. Please make sure that configuration parameter is deployed before deploying this ;-)

File: app/models/user.rb (hunk 1 above)
Comment (javitonino) [inline app/models/user.rb]: Thanks for mentioning this. :heart: it!

**Comment by javitonino** [comment]
Shouldn't need to deploy any configuration changes, if the parameter is not found it defaults to true, as was doing before.

Anyway, the config change will go first :+1:

**Comment by juanignaciosl** [comment]
Oh, I didn't realize. Since its name is `gravatar_enabled` I thought that default was `false`, but it makes sense keep current behaviour 👍

**Comment by javitonino** [comment]
New flag deployed, merging this...

**Pull request merged**